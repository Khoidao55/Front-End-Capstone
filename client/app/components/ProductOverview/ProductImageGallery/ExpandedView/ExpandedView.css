.expanded-image {
  margin: auto;
  display: flex;
  width: 80%;
  max-width: 700px;
  z-index: 30;
}

.expanded-product-thumbnail {
  display:flex;
  margin-left: -100px;
  margin-top: 50px;
  position: absolute;
}