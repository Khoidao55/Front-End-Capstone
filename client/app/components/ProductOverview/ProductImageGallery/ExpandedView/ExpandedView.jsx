import React from 'react';

const ExpandedView = () => (
  <div>
    <h1>Hello</h1>
  </div>
)

export default ExpandedView;
