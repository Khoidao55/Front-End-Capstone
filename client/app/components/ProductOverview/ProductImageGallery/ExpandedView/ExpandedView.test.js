import React from 'react';
import { shallow } from 'enzyme';
import ExpandedView from './ExpandedView.jsx';

describe('Product Image Expanded View', () => {
  test('should render my component', () => {
    const wrapper = shallow(<ExpandedView />);

    expect(wrapper).toMatchSnapshot();
  });
});
