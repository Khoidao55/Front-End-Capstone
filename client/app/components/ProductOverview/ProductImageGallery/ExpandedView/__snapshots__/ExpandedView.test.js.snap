// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Image Expanded View should render my component 1`] = `ShallowWrapper {}`;
